frontend/viewer/src/lib/components/ui/resizable/resizable-handle.svelte

```diff
@@ -3,28 +3,94 @@
   import type {WithoutChildrenOrChild} from 'bits-ui';
   import * as ResizablePrimitive from 'paneforge';
   import {Icon} from '../icon';
+  import Button, {type ButtonProps} from '../button/button.svelte';
+  import {Debounced} from 'runed';
+  import {type IconClass} from '$lib/icon-class';
+  import {onMount, tick} from 'svelte';
 
   let {
     ref = $bindable(null),
     class: className,
     withHandle = false,
+    leftPane,
+    rightPane,
     ...restProps
   }: WithoutChildrenOrChild<ResizablePrimitive.PaneResizerProps> & {
     withHandle?: boolean;
+    leftPane?: ResizablePrimitive.Pane;
+    rightPane?: ResizablePrimitive.Pane;
   } = $props();
+
+  let dragging = $state(false);
+  const draggingDebounced = new Debounced(() => dragging, 300);
+
+  type OnClickHandler = ButtonProps['onclick'];
+  const leftResizerOnClick = $derived.by(() => getPaneResizerOnClick(leftPane, rightPane));
+  const rightResizerOnClick = $derived.by(() => getPaneResizerOnClick(rightPane, leftPane));
```

Comment:
why not `$derived(getPaneResizerOnClick(rightPane, leftPane))`? It does the same thing.